source/SquadLeader/app/app.iml

```diff
@@ -24,7 +26,7 @@
       </configuration>
     </facet>
   </component>
-  <component name="NewModuleRootManager" inherit-compiler-output="false">
+  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_6" inherit-compiler-output="false">
```

Comment:
Android Studio told me it was upping the language level from 5 to 6 and I didn't argue. I can research if you're concerned.